react-trainning/src/components/Dice/Dice.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react'
+import Dice1 from './../../../public/img/dice1.png'
+import Dice2 from './../../../public/img/dice2.png'
+import Dice3 from './../../../public/img/dice3.png'
+import Dice4 from './../../../public/img/dice4.png'
+import Dice5 from './../../../public/img/dice5.png'
```

Comment:
Instead of /public/img/dice3.png, use /img/dice3.png.
files in the public directory are served at the root path.